ruby/google-protobuf.gemspec

```diff
@@ -42,5 +44,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake-compiler", "~> 1.1.0"
   s.add_development_dependency "test-unit", '~> 3.0', '>= 3.0.9'
   # bigdecimal must be used as a non-built in gem as of ruby-3.4
-  s.add_development_dependency "bigdecimal"
+  s.add_development_dependency "bigdecimal" if RUBY_PLATFORM >= '3.4.0'
```

Comment:
It should be `RUBY_VERSION`?
![image](https://github.com/user-attachments/assets/c1f699ba-8cf7-446f-a5da-cabe5e0f6b0b)